### ISSUE TITLE
[Website update] Community page

### ISSUE THREAD

**Issue by krisfreedain** [opensearch-project/project-website#1177]
**Is your feature request related to a problem? Please describe.**
the website currently lacks a central place to display all things happening across the OpenSearch community. 

**Describe the solution you'd like**
Community page to highlight different activities across the OpenSearch community. Highlights include 

- Events page
- OpenSearchCon page
- Where we’ll be - external cons we’re attending 
- Social
- User Groups (soon)

**Comment by nateynateynate** [comment]
@krisfreedain : Would you consider this completed to your satisfaction with the newly added 'Community' dropdown?

**Comment by krisfreedain** [comment]
@nateynateynate  No. There are a lot of additional pieces that would belong on a proper 'Community' page. We spoke about it a few times on the weekly web meeting, but I should draw up better requirements for it.
